zerver/models.py

```diff
@@ -680,6 +687,14 @@ def webathena_enabled(self) -> bool:
     def presence_disabled(self) -> bool:
         return self.is_zephyr_mirror_realm
 
+    @property
+    def oss_showcase(self) -> int:
+        return self.oss_showcase_policy
+
+    @property
+    def get_plan_type(self) -> int:
+        return self.plan_type
```

Comment:
Let's remove these properties -- they're confusing; code should just be accessing the actual data model properties.